services/apis/fence/fenceSequences.js

```diff
@@ -112,5 +113,60 @@ module.exports = {
   async deleteFile(guid) {
     const res = await fenceTasks.deleteFile(guid);
     fenceQuestions.assertStatusCode(res, 204);
-  }
+  },
+
+
+  /**
+   * Cleans up fence's DBs for links and service accounts
+   * Takes the google and users utils as params
+   * @returns {Promise<void>}
+   */
+  async suiteCleanup(google, users) {
+    // unlock the lockable google project
+    await google.unlockGoogleProject(fenceProps.googleProjectDynamic);
```

Comment:
shouldn't you get the response from `unlockGoogleProject` and make sure it's true?

Reply:
actually - we can't assert here, since we don't want it to throw if the lock belongs to another session